### PR TITLE
PA pre migration work

### PR DESCRIPTION
**Pull request by JessicaWNava**
This PR implements the pre migration work for PA. 

Test Steps:
1. Review that the changes in organizations.yml match PA previous settings
2. Check that the resulting message matches the outbound message from the Covid pipeline
3. Make sure the tests pass

## Changes
- Added settings for PA to organizations.yml
- Created the PA receiver transform
- Added an integration test for PA

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [x] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/PA/PA-receiver-transform.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  - name: pa-receiving-application-namespace-id
+    value: [ '"PA-ELR"' ]
+    hl7Spec: [ '%{MSH}-5-1' ]
+
+  - name: pa-receiving-facility-namespace-id
+    value: [ '"PADOH"' ]
+    hl7Spec: [ '%{MSH}-6-1' ]
+
+  - name: pa-sending-facility-namespace-id
```
Comment (oslynn) [inline prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/PA/PA-receiver-transform.yml:38]: Where do you see these values to be filled?  Is it in the original message?

File: prime-router/settings/STLTs/PA/pa-phd.yml [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides:
```
Comment (oslynn) [inline prime-router/settings/STLTs/PA/pa-phd.yml:47]: Are we going to implement this valueSetOverrides in receiver transform?

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/PA/PA-receiver-transform.yml (hunk 1 above)
Comment (JessicaWNava) [reply prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/PA/PA-receiver-transform.yml:38]: Yes

File: prime-router/settings/STLTs/PA/pa-phd.yml (hunk 2 above)
Comment (JessicaWNava) [reply prime-router/settings/STLTs/PA/pa-phd.yml:47]: I asked Victor and he said these were done here.

Comment (JessicaWNava) [reply prime-router/settings/STLTs/PA/pa-phd.yml]: Fixed and tested, ready for re-review @oslynn

**Review by victor-chaparro — APPROVED** [review]

**Review by oslynn — APPROVED** [review]

**Pull request merged**